README.md

```diff
@@ -66,67 +93,27 @@ You are now ready to start writing some code! If you dont already have a favorit
 
 https://atom.io/
 
-Regardless of what editor you are using you will want it to be ready for coding a react / node javascript application so you will want plugins related to syntax highlighting and inline helpers for those languages. In atom these are principally:
```

Comment:
removed some of this editor stuff - I'm not sure freshmen will know what a linter is, and it felt like we might be overwhelming them with tools and things to research

Reply:
Yeah Im assuming these guys will have _some_ kind of editor setup. We can deal with that as we go.